aep/general/0159/aep.md.j2

```diff
@@ -1,5 +1,69 @@
 # Reading across collections
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/13) for more
-information.
+Sometimes, it is useful for a user to be able to retrieve resources across
+multiple collections, or retrieve a single resource without needing to know
+what collection it is in.
+
+## Guidance
+
+APIs **may** support reading resources across multiple collections by allowing
+users to specify a `-` (the hyphen or dash character) as a wildcard character
+in a standard [`List`](./list) method:
+
+```
+GET /v1/publishers/-/books?filter=...
+```
+
+- The URI pattern **must** still be specified with `*` and permit the
+  collection to be specified; a URI pattern **must not** hard-code the `-`
+  character.
+- The method **must** explicitly document that this behavior is supported.
```

Comment:
can we add an annotation in addition? a human-readable docstring isn't amenable to generated clients.

Reply:
No objection from me, I like lots of annotations.  Filed [a ticket](https://github.com/aep-dev/aep.dev/issues/169) so as not to block this PR on another PR creating the annotation.